docs/source/user_guide/types.rst

```diff
@@ -0,0 +1,145 @@
+======================
+Type Support In Koalas
+======================
+
+.. currentmodule:: databricks.koalas
+
+
+This is based on Koalas 1.1.4.
```

Comment:
I don't think we need this.
If we change the type support, we should also update this doc.

Reply:
Sounds reasonable. Thanks!!